README.md

```diff
@@ -1,109 +1,218 @@
+   
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>My portfolio</b></h3>
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Portfolio-setup-and-mobile-first] <a name="about-project"></a>
+# 📖 [portfolio] <a name="about-project"></a>
+
+**portfolio** is my web site where I show case my project ...
+
+## 🛠 Built With <a name="built-with"></a>
 
-> This is a Microverse Portfolio Project. I have created it with Mobile First approach. I have created it with HTML CSS . In CSS I used mainly FlexBox for layout to make it responsive.
+### Tech Stack <a name="tech-stack"></a>
 
-**[Portfolio-setup-and-mobile-first]**
- is a first portfolio project at microverse 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+    <li><a href="#">css</a></li>
+    
+  </ul>
+</details>
 
-## 🛠 Built With 
-<a name="built-with">html</a>
-<a name="built-with">css</a>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **It show case my project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> https://massazoum.github.io/Portfolio-setup-and-mobile-first/
+- [Live Demo Link](https://massazoum.github.io/Portfolio-setup-and-mobile-first)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
-Run git clone https://github.com/massazoum/Portfolio-setup-and-mobile-first.git
+### Prerequisites
+
+In order to run this project you need:
+- vs code 
+- github
+- laptop
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
```

Comment:
- [ ] As the previous reviewer said, Kindly ensure every statement is complete and personalized 😉 incomplete statements will give your project a less professional look😉😉